monai/apps/pathology/handlers/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from monai.config import KeysCollection
+from monai.utils import ensure_tuple
+
+
+def from_engine_hovernet(keys: KeysCollection, nested_key: str):
+    """
+    Utility function to simplify the `batch_transform` or `output_transform` args of ignite components
```

Comment:
This could document the reasons why it's needed for hovernet. The other docstring could be omitted in my opinion.

Reply:
I agree. I think we can refer to `from_engine` and only mention how/why it is different but I like to keep the example too as it requires additional argument.

Reply:
Hi @wyli, thanks for your opinion, have updated it in the latest commit.